source/chapters/user_interface.rst

```diff
@@ -1168,3 +1135,36 @@ The features in detail:
 
 .. seealso:: For more information, go to the chapter
              :ref:`library-previewing-tracks`.
+
+.. _interface-expansion-buttons:
+
+Section Expansion Buttons
+=========================
+
+  .. figure:: ../_static/2.2/expansion-buttons-top.png
+   :align: center
+   :width: 430px
+   :figwidth: 90%
+   :alt: The section expansion buttons
+   :figclass: pretty-figures
+
+   The section expansion buttons on top
+
+  .. figure:: ../_static/2.2/expansion-buttons-skin-settings.png
+    :align: center
+    :width: 245px
+    :figwidth: 90%
+    :alt: The section expansion buttons
+    :figclass: pretty-figures
+
+    The section expansion options under the Skin settings on the right.
+
+On top, and on the right side of the user interface are the section expansion buttons and options.
+
+If you click on either :guilabel:`EFFECTS`, :guilabel:`SAMPLERS` or :guilabel:`MICS & AUX` then you will enable control sections for interacting with:
+
+* :ref:`interface-effects`
+* :ref:`interface-sampler`
+* :ref:`interface-mic`
+
+You can also use the specific :ref:`appendix-shortcuts`.
```

Comment:
Let's move that below the GUI quick toggles image/desription.

Reply:
Ok, I will work on this right away.

Reply:
Done